src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1647,7 +1647,10 @@ private void UpdateConditionDifferencesBetweenPatches(Patch selectedPatch, Patch
         }
     }
 
-    private void UpdateShownPatchDetails()
+    /// <summary>
+    ///   Updates the patch details specific to the Patch Map tab
+    /// </summary>
+    private void UpdatePatchMapDetails()
```

Comment:
Why the rename here? This still acts on just the one patch, and not the entire map, which is what immediately came to mind when I saw this rename.